.npmignore

```diff
@@ -1,4 +1,4 @@
-examples
```

Comment:
I replaced the examples with tests. The examples where not extensive enough to be useful. The test will start `eslint` in a new child process to make sure the config is valid.